builder/vmware/common/driver_esxi.go

```diff
@@ -747,12 +748,18 @@ func (d *EsxiDriver) connect() error {
 		auth = append(auth, gossh.PublicKeys(signer))
 	}
 
+	knownHostsFile := filepath.Join(os.Getenv("HOME"), ".ssh", "known_hosts")
+	hostKeyCallback, err := knownhosts.New(knownHostsFile)
+	if err != nil {
+		return err
```

Comment:
Looking at this, this means that if I don't have a local hosts file the builder will fail?
Since Packer implements its own SSH code instead of relying on OpenSSH or alternatives, it is conceivable that host machines won't have one, so we might break their builds by merging this I'm afraid